### ISSUE TITLE
what 's means about "not found: .......THNN.lua:108: NYI: call arg type	"  ?

### ISSUE THREAD

**Issue by SeekPoint** [abhshkdz/neural-vqa#13]
rzai@rzai00:~/prj/neural-vqa$ CUDA_VISIBLE_DEVICES=0 th train.lua -gpuid 0
{
  beta : 0.999
  train_fc7_file : "data/train_fc7.t7"
  embedding_size : 512
  val_fc7_file : "data/val_fc7.t7"
  dropout : 0.5
  seed : 981723
  data_dir : "data"
  learning_rate_decay : 0.95
  epsilon : 1e-08
  learning_rate_decay_after : 15
  max_epochs : 50
  checkpoint_dir : "checkpoints"
  rnn_size : 512
  alpha : 0.8
  num_layers : 2
  init_from : ""
  savefile : "vqa"
  batch_size : 2
  val_fc7_image_id_file : "data/val_fc7_image_id.t7"
  train_fc7_image_id_file : "data/train_fc7_image_id.t7"
  save_every : 1000
  learning_rate : 0.0004
  gpuid : 0
}
not found: THNN_CudaHalfELU_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfELU_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfHardTanh_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfHardTanh_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfLeakyReLU_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfLeakyReLU_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfLookupTable_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfLookupTable_renorm/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfMarginCriterion_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfMarginCriterion_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfMultiMarginCriterion_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfMultiMarginCriterion_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfPReLU_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSpatialConvolutionLocal_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSpatialConvolutionMM_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSpatialCrossMapLRN_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSpatialCrossMapLRN_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSpatialDilatedConvolution_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSpatialFullConvolution_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSoftPlus_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSoftPlus_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSoftShrink_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSoftShrink_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfSqrt_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfTemporalConvolution_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfThreshold_updateOutput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfThreshold_updateGradInput/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfVolumetricConvolution_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfVolumetricDilatedConvolution_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
not found: THNN_CudaHalfVolumetricFullConvolution_accGradParameters/home/rzai/torch/install/share/lua/5.1/nn/THNN.lua:108: NYI: call arg type	
using CUDA on GPU 0...	
Loading data files...	
Loading train fc7 features from data/train_fc7.t7

**Comment by abhshkdz** [comment]
Ouch. Seems like an issue with Torch installation. Could you try `rm -rf /home/rzai/torch` and reinstalling Torch as per instructions given here: https://github.com/torch/distro#lua-and-torch.

**Comment by SeekPoint** [comment]
acturally, to avoid the memory issue,I use 
 TORCH_LUA_VERSION=LUA51 ./install.sh
to install lua, but I didn't reinstall other package like nn,rnn

**Comment by abhshkdz** [comment]
I see. And does it work fine after reinstalling nn?

**Comment by SeekPoint** [comment]
so far, it works well without reinstall nn etc

i will try later to find whether the reinstallation of nn,rnn can remove the bad message
